coralproject/talk
# Don't Agree flags

## What does this PR do?

Moves "Don't Agree" flags to their own datatype and removes them from the moderation queue.

## How do I test this PR?

1. Report a comment.
2. Select "don't agree" as a reason.
3. Flag on the report button should turn red.
4. Reload.
5. Flag on the report button should still be red.
6. Click the admin panel.
7. The "don't agree" flag should not appear the flagged comments section.

## Note
There's a lot of boilerplate required to add a new action type at this point, we should think about creative ways to minimize it for plugin developers.
